.github/workflows/PR.yml

```
@@ -11,7 +11,7 @@ jobs:
     name: Validate
     runs-on: ubuntu-latest
     steps:
-      - uses: amannn/action-semantic-pull-request@v5.1.0
+      -  uses: amannn/action-semantic-pull-request@v5.2.0
```

I think this space needs to be removed
```suggestion
      - uses: amannn/action-semantic-pull-request@v5.2.0
```

---

Thank you!